shared/elements/gaia_grid/style.css

```diff
@@ -374,3 +374,9 @@
 .edit-mode.hover-over-top {
   transform: translateY(0.8rem);
 }
+
+/* RTL View */
+:-moz-dir(rtl) .group .header .toggle {
+  left: 0;
+  right: unset;
```

Comment:
Interesting, wasn't aware of this syntax :) (or is it that any non-number will cause it to be unset?)